pom.xml

```diff
@@ -160,7 +160,7 @@ The MIT License (MIT)
                         <limit>
                           <counter>BRANCH</counter>
                           <value>COVEREDRATIO</value>
-                          <minimum>0.66</minimum>
+                          <minimum>0.65</minimum>
```

Comment:
Was this part of the original request? Just asking to make sure this change will not break anything else.

Reply:
@allanws I think they are not a proper solution to this problem, because Zip it's a record based solution and not work properly with streams (there is GZip to work with streams). So, I'll reformulate completely this part. But thanks for your review!

Reply:
I meant about the jacoco settings that were changed. I'm not sure how this will affect the entire project.